server/helpers/auth.js

```diff
@@ -0,0 +1,5 @@
+module.exports = (req, res, next) => {
+  if(req.isAuthenticated()) return next;
+  // Send user back to login page
+  res.redirect('/');
```

Comment:
I'm not sure this is correct, we're currently doing a REST interface so all our routes return a JSON and expect a JSON.

Look into sending a 401 status code, with maybe a json with details on the error